CriteoPublisherSdk/Tests/UnitTests/CR_BidManagerFeedbackTests.m

```diff
@@ -293,7 +293,41 @@ - (void)testFeedbackMessageStateOnEmptyBid {
   XCTAssertEqualObjects(message, expected);
 }
 
-#pragma mark - Ready to send management
+#pragma mark - Feedback Message State (Live bidding)
+
+- (void)testFeedbackMessageStateOnLiveBidConsumed {
+  CR_FeedbackMessage *expected = [self.defaultMessage copy];
+  expected.cdbCallEndTimestamp = self.dateInMillisecondsNumber;
+  expected.elapsedTimestamp = self.dateInMillisecondsNumber;
+  expected.cachedBidUsed = YES;
+  expected.zoneId = self.validBid.zoneId;
+
+  self.cdbResponse.cdbBids = @[ self.validBid ];
+  [self prefetchBidWithMockedResponseForAdUnit:self.adUnit];
```

Comment:
I think you don't need to prefetch for live bidding

Reply:
This method is used to enqueue a metric that will be sent with next live bid request.
I agree prefetching isn't the most logical thing in these tests, but the goal is in fact to prepare CSM queue state.
Also, that comes from legacy tests, which has the comment on `prefetchBidWithMockedResponseForAdUnit`:
```// TODO: improve tests relying on this so that we don't need to call prefetchBid.```

Reply:
Ok